client/sandbox/react-nextjs/pages/play/linkunlink_ruleparams.tsx

```diff
@@ -0,0 +1,203 @@
+import { useState } from 'react';
+import { init, tx, id, i } from '@instantdb/react';
+import config from '../../config';
+import EphemeralAppPage from '../../components/EphemeralAppPage';
+
```

Comment:
cc @tonsky made another example with ruleparams -- thought this would be nice, since we can more quickly test "acting" as different users.